packages/aws-cdk/test/toolkit/cli-io-host.test.ts

```diff
@@ -226,24 +226,6 @@ describe('CliIoHost', () => {
     });
   });
 
-  describe('error handling', () => {
```

Comment:
This test was testing the uneccessary callback handling. It only passes because we mock the stdout implementation to invoke the callback. In reality the callback never gets invoked. So really this only tested that the if statement inside the callback worked as expected.